app/helpers/articles_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ArticlesHelper
-  def hasVid(article)
+  def has_vid?(article)
```

Comment:
haha now that you're already on this you might go an extra mile and call the method "has_video?"

Don't know if there's a conflict but two letters make the method more self explanatory.

Reply:
Yeah, good call. No conflict.

Looking at this, I know I wrote this method originally when this was a solo project (our public git blame doesn't preserve the whole history so I'm off the hook!)

No clue what I was thinking with this name.